apps/nextjs-website/src/editorialComponents/BannerLink/BannerLink.tsx

```diff
@@ -19,6 +19,13 @@ export const BannerLink = (props: BannerLinkProps) => {
   const backgroundColor =
     theme === 'dark' ? palette.primary.dark : palette.primary.light;
 
+  const bodyElement =
+    typeof body === 'string' ? (
+      <div dangerouslySetInnerHTML={{ __html: DOMPurify.sanitize(body) }} />
```

Comment:
Instead of `dangerouslySetInnerHTML` you should use something like this:
```  const legalInfo = t.rich('legalInfo', {
    strong: (chunks) => <strong>{chunks}</strong>,
    br: () => <br></br>,
  }) as string;``` (see `SiteFooter.tsx`)

Reply:
My bad, this is not a translation but a content of these types `string | JSX.Element`. Using DOM sanitazer is the right way.